AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -6053,6 +6053,8 @@ com-magazin.de##w-div[role$="dialog"]
 ||mediaset.it/include/js/attivaPopUpEstero.js
 ||muzofond.com/mithdl.js
 ||sud.ua/js/jquery.arcticmodal*.min.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/176486
+||assets.zephr.com/zephr-browser/*/zephr-browser.umd.js^$script,domain=tampabay.com
```

Comment:
Can be added here
![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/46bee943-845a-47e5-aeba-2c612c6c6534)

`^` and `script` are not required here.

Reply:
if everything is true, moved to the correct place, thanks